packages/patternfly-4/react-table/src/components/Table/Table.js

```diff
@@ -226,7 +226,7 @@ class Table extends React.Component {
           {...props}
           renderers={{
             body: {
-              wrapper: bodyWrapper || BodyWrapper,
+              wrapper: bodyWrapper(rows) || BodyWrapper,
```

Comment:
@suomiy - if i'd like to filter rows, what is the proper way of doing so w/ this extension model?  Currently I am passing in rows... will likely have to refactor this bit...
https://github.com/patternfly/patternfly-react/pull/1250/files#diff-82199de5c757484a78f5c7f625f15843

Reply:
We should try to aim at chaining all the components with HoC principles as implemented in `react-inline-edit-extension`. `react-table` should be the most bottom component. We should add necessary basic functionality which  `react-virtualized-extension` might miss to `react-table` and activate it by  each extension. I  will discuss the details with you offline.

Reply:
thanks! just wanted to confirm w/ you first what the API should be. Happy to adjust this piece.

Reply:
Refactored `BodyWrapper` to use a similar API as in `react-inline-edit-extension`. It is clear to the see difference now. The `react-virtualized-extension` needs to actually override rendering of received `rows` b/c it tracks scroll position with the `start-row`, `end-row` and `extra-row`. These `tr` 's are just used to tracking purposes and are based on the implementation in `reactabular-virtualized`. I have marked them `aria-hidden` so that screen readers ignore them, but they are just used for tracking purposes. They could technically be used for test cases as well as scroll position is adjusted.

Reply:
`Body` is also similar, however it needs to calculate the currently rendered rows, thus the `onScroll` is extended to do this and `rowsToRender` are passed down as a prop.

Reply:
Not sure if it is necessary at this testing stage, but again I think the extension should be implemented as HoC in the future to allow chaining of multiple extensions.

Reply:
i've started this in https://github.com/priley86/patternfly-react/commit/2f73d45129af476faa565c4848e3f623d27b740a but noted some issues, will have to see if this possible or not. At the moment, virtualized is not targeted towards supporting other table features which make further modifications to default row renderings/row heights (i.e. collapse/inline-edit/etc.). This further convolutes row height calculation logic and I believe that making use of server pagination/infinite scroll is better for that use case. Virtual scroll is a simple helper intended to emulate react-virtualized (virtual scroll) allowing the user to simply pass all rows and have a utility calculate visible offsets.